lib/core/src/sdk.rs

```diff
@@ -1350,6 +1388,16 @@ impl LiquidSdk {
             compressed: true,
             inner: refund_keypair.public_key(),
         };
+        let webhook = self.persister.get_webhook_url()?.map(|url| ChainWebhook {
+            url,
+            hash_swap_id: Some(true),
+            status: Some(vec![
+                ChainSwapStates::TransactionConfirmed,
```

Comment:
If this is the user lockup then again perhaps we don't need such notification?

Reply:
Nothing is done but record the Bitcoin user_lockup_tx_id, we would need to add an alternative way to record this

Reply:
perhaps we need to fetch it onchain if the state is advanced and we missed it?

Reply:
Now thinking even without the notification feature we should take care of the case where we missed the TransactionConfirmed event as use can be offline while this happens. so if we need the user lockup tx id then we need to fetch it when is needed.